src/lustre/lustreExpr.ml

```diff
@@ -1905,46 +1772,23 @@ let eval_to_int expr =
         (Numeral.of_big_int
            (Decimal.to_big_int
               (Symbol.decimal_of_symbol s)))
-    | Term.T.Const s when Symbol.is_ubv8 s ->
-      Term.mk_num
-        (Bitvector.ubv8_to_num
-          (Symbol.ubitvector_of_symbol s))
```

Comment:
This code performs a conversion from an 8-bit unsigned integer literal (a known concrete value) to an integer, instead of delegating the conversion to the solver. We should retain this optimization for arbitrary bit widths.